playwright.config.ts

```diff
@@ -1,6 +1,6 @@
-import { defineConfig, devices } from '@playwright/test';
-import * as dotenv from 'dotenv';
-dotenv.config({ path: './e2e/envs/.env.playwright' });
+import { defineConfig, devices } from "@playwright/test";
```

Comment:
This is a minor issue, but is it possible to set ignore so that double-quote is not set in the config.ts file? The default setting for WebUI projects is single quote.